cmd/admin-revoker/main.go

```diff
@@ -330,7 +328,7 @@ func (r *revoker) blockByPrivateKey(ctx context.Context, comment string, private
 		return err
 	}
 
-	spkiHash, err := getPublicKeySPKIHash(publicKey)
+	spkiHash, err := core.KeyDigest(publicKey)
```

Comment:
`core.KeyDigest` returns a `core.Sha256Digest` which is a `[sha256.Size]byte`. You could replace all usage of `spkiHash[:]` with just `spkiHash`. Some method signatures would need to change to take `core.Sha256Digest` instead of `spki []byte`.

*Edit:* Upon further thought, you'll eventually end up with a few test errors such as the following which seems like a pain.
```
--- FAIL: TestBlockAndRevokeByPrivateKey (0.37s)
    main_test.go:154: countCertsMatchingSPKIHash for dupe failed: failed to select one blockedKeys: sql: converting argument $1 type: unsupported type core.Sha256Digest, a array
```

Reply:
Yeah, I like this approach better. It greatly reduces the number of places I have to use the `[:]` operation to turn the core.Sha256Digest into a generic byte slice, and gets a little bit of self-documenting type safety at the same time. Done.